extensions/2.0/Khronos/KHR_avatar_skeleton_bindpose/README.md

```diff
@@ -0,0 +1,119 @@
+# KHR_avatar_skeleton_bindpose
+
+## Contributors
+
+- Ken Jakubzak, Meta
+- Hideaki Eguchi / VirtualCast, Inc.
+- K. S. Ernest (iFire) Lee, Individual Contributor / https://github.com/fire
+- Shinnosuke Iwaki / VirtualCast, Inc.
+- 0b5vr / pixiv Inc.
+- Leonard Daly, Individual Contributor
+- Nick Burkard, Meta
+
+## Status
+
+**Draft** – This extension is not yet ratified by the Khronos Group and is subject to change.
+
+## Dependencies
+
+Written against the glTF 2.0 specification.  
+Dependent on: `KHR_avatar`
+
+## Overview
+
+**THIS EXTENSION IS VERY MUCH WORK IN PROGRESS**
+
+The `KHR_avatar_skeleton_bindpose` extension defines a canonical bind pose for the avatar's skeleton. This pose serves as a neutral reference that supports animation retargeting, mesh deformation validation, and consistency across toolchains. It is particularly useful when the glTF `skins[].inverseBindMatrices` are insufficient or absent for expressing the true skeletal rest position.
+
+## Extension Schema
+
+```json
+{
+  "extensions": {
+    "KHR_avatar_skeleton_bindpose": {
+      "poseType": "TPose",
+      "skeleton": 0,
+      "jointBindPoses": [
+        {
+          "joint": 0,
+          "matrix": [
```

Comment:
There's likely a better way to represent this via an additional skin object with a different skeleton and InverseBindMatrices reference for the A/T/Custom Pose.